### DISCUSSION TITLE
Can't load codespaces in firefox when `privacy.resistFingerprinting=true`

### DISCUSSION THREAD

**Discussion by ktmzcpl** [community/community#42091] [Codespaces]
### Select Topic Area

Bug

### Body

If the `privacy.resistFingerprinting` is been set to `true` in firefox (this is what happend in Tor browser, but it also can be done in a normal firefox), we can't load codespaces.
It throws an error: ` r is undefined`.

**Comment by jaysee260** [comment]
Hey @ktmzcpl 👋 ! Thanks for reaching out. I'm sorry you're having trouble using Codespaces in your browser of choice 😔. I have been able to reproduce this myself, and have raised the issue internally so we can take a look into it.

Do you use VS Code? In the meantime, I might suggest using it to connect to your codespaces. It looks like while the web view won't load on Firefox when that setting is enabled, the codespace still gets created.

If you go to [github.com/codespaces](https://github.com/codespaces), you can see all your codespaces listed. To the very right of a given codespace, you'll see a button with 3 dots. Clicking on it should bring up a menu. In that menu, there is an `Open in ...` option. Clicking on that option should reveal a few more options, among them is one that says `Open in Visual Studio Code`.

Again, sorry for the trouble! We'll look into it 🔍.

![image](https://user-images.githubusercontent.com/33580723/209004990-580f5ac8-7404-44a0-89dd-f278d16ff9c1.png)

![image](https://user-images.githubusercontent.com/33580723/209005112-2ae84a74-25c6-4e06-b9f0-be618b1e5877.png)

**Comment by mjbear** [comment]
This might be helpful.
#124964 - [specifically this comment](https://github.com/orgs/community/discussions/124964#discussioncomment-9802403)
